examples/medplum-task-demo/src/pages/NotesPage.tsx

```diff
@@ -0,0 +1,58 @@
+import { Blockquote, Stack } from '@mantine/core';
+import { Annotation, Task } from '@medplum/fhirtypes';
+import { Document } from '@medplum/react';
+
+export interface NotesPageProps {
+  task: Task;
+}
+
+export function NotesPage(props: NotesPageProps): JSX.Element {
+  const notes = props.task.note;
+
+  // Sort notes so the most recent are at the top of the page
+  const sortedNotes = sortNotesByTime(notes);
+
+  // Display if the task does not have any notes
+  if (!sortedNotes) {
```

Comment:
Minor recommendation:  change `sortNotesByTime` to require notes, and move this check to line 11 `if (!notes)`.  You can sidestep a bunch of the `undefined` checks that way

Reply:
Makes sense, this is updated